### DISCUSSION TITLE
必读：关于最终清理用户免费额度的说明和公告

### DISCUSSION THREAD

**Discussion by BennyThink** [BurnHair/docs#171] [网站公告]
# TLDR
北京时间8月3日零点，将会清空所有**免费用户**的免费额度。所有用户的奖励额度、付费额度、付费用户的任何额度均不受此变动的影响

# 缘起
一切的一切，都源自于这张帖子《不要再被头顶冒火欺骗了，用始皇的 API 服务吧》。

当时我们在这件事情上讨论如何处理，最终得出的结论是**转变产品策略**。当时我们进行了一系列不同意见的交换，最终我独裁式的选择缓和的处理方式：逐步处理而不是一刀切的清空免费额度

# 清零计划
昨天进行了一些未活跃用户的清理，具体清理规则可以看这里 #167 

结果第二天就收到了20多封邮件，还有别人发给我的更让人上火的[某帖子](https://linux.do/t/topic/157312/)😂 （这个帖子被删了，当时大意了忘记存档 不过嘛总有类似[让人上火](https://web.archive.org/web/20240725043249/https://linux.do/t/topic/158667)的东西）

# 错误
在三个星期前做下转型的决定时，我就意识到了，最后无论怎么转型，都是我的错，都会影响站点的口碑和声誉。

* [用户协议](https://burn.hair/tos)中明确指出，重复注册会被无另行通知的情况下封禁，还是有人说这样做过分
* 用户协议中说，保留回收公益额度的权利，还有人说恶心人
* 清零前会写一个帖子，提前几天时间预告，还有人说“说改就改” 
* ……

我意识到，即使点了同意，也没有人读用户协议的。

# 长痛不如短痛

<img width="712" alt="image" src="https://github.com/user-attachments/assets/8506bb10-5d23-4248-9848-f64a979d2422">

这个图是大概是最近几天的消耗，付费用户消耗 259，免费用户消耗662，也就是说我拿了70%的资源用来做可能吃力不讨好的事情。

这对于付费用户来说很不公平，而且我也不好保证在这种极度不均衡的情况下，保证付费用户的利益。

所以真不如当时听了投资人的意见，长痛不如短痛，一刀切最好。

> 这个才是重要点的事，三个月都转化不了的用户早点清退掉，给付费用户极致的体验
> 既然公益影响稳定性，就不搞了，很简单

随便你们怎么说了，说是我拿这个那个帖子做借口也好，说我在怪罪、下台阶、收割韭菜、玩不起等等都好。无所谓，就当我是真的这样想的，就当我觉得猪养肥了要杀了。

# 最终的决定是
1. 一周左右的通知期，避免有人说时间太短
1. 北京时间8月3日零点（恰巧那个帖子一个月），所有**非付费用户**的免费额度将会被清空
2. 付费用户不会受到任何影响，奖励额度也不会有影响
3. 所有和这次清零相关的问题，包括邮件、私聊等不会回复
4. 签到也将改变为只支持付费用户
5. 用户协议推出一月有余，届时所有未同意用户协议的用户都无法使用本服务，只要登录同意即可恢复
6. 如果额度分组时有错，请在通知期结束之前告知我查询修改
7. 所有来自该黑产论坛的用户都将无法跳转访问本站（根据referer判断）

**Comment by anonymouswhac** [comment]
免费也用的挺久的了，虽然我用的少，持续签到也到了26$。
看了前一篇文章，居然有人恶意的去刷额度。
亏本的买卖没法持续，承蒙这么久以来一直免费使用。
希望站长早日盈利，另外反馈一下，虽然清空了，但是免费账户好像还是可以签到。

`恭喜您获得了2142的额度，您的总额度为13423913 - $26.85！（清空前）`
`恭喜您获得了42881的额度，您的总额度为42881 - $0.09！（清空后仍然可以签到）`

**Comment by sandyplus** [comment]
有没有充值1-2刀的商品？最少冲70多，充那么多用不完啊！

**Reply by BennyThink** [reply]
都点完 会自动刷新的

**Reply by NoahBishop** [reply]
> 都点完 会自动刷新的

目前刷新好几次，最低看见个3$

**Comment by CainSu** [comment]
从来都觉得，任何一个服务不盈利，纯为爱发电，必定走不长远。虽然用得不多，也是在很早之前就已经充值，希望能早日盈利提供更稳定廉价的服务。

**Comment by Loongphy** [comment]
升米恩，斗米仇。
![image](https://github.com/user-attachments/assets/0fea57a8-c82b-4eaa-9d83-76fa8b278919)

**Comment by pryyxj** [comment]
做的有点过了吧
![image](https://github.com/user-attachments/assets/4c38e269-b461-4d5b-a85f-307c20e9f28d)

**Reply by yangzhiec7** [reply]
支持你

**Reply by hengyu616** [reply]
所以你到底站那边，刚开始还挺蒙，后来免费清空才哈哈大笑

**Comment by ghost** [comment]
真是让人贻笑大方，短视又高傲的产品运营
https://github.com/orgs/BurnHair/discussions/144
存档：https://archive.ph/ijeQZ

**Reply by BennyThink** [reply]
感谢批评。在运营初期我们确实犯了很大的错误，短视且高傲的批评我们认为是合理的。不过也没必要存档，我不会删帖子的。

**Reply by ghost** [reply]
> 感谢批评。在运营初期我们确实犯了很大的错误，短视且高傲的批评我们认为是合理的。不过也没必要存档，我不会删帖子的。

“黑产论坛”事后更新：https://archive.ph/KsN5v

**Comment by pryyxj** [comment]
始皇官方认证
![image](https://github.com/user-attachments/assets/16edacc5-b6a5-48c5-bee6-eb8c1ee14cfa)

L站邀请码一个：http去s:/除/linux汉.do字/invi打tes/j开1h6nBaEyg

**Reply by Passerby1011** [reply]
https://linux.do/invites/j1h6nBaEyg

**Reply by pryyxj** [reply]
> https://linux.do/invites/j1h6nBaEyg
没啦，用完了

**Reply by MdzzSakura** [reply]
https://linux.do/invites/Q1wLfDiwks

**Comment by cn1860** [comment]
支持清理，不要关站，还要用呀。竞品诋毁就诋毁吧，反正堵不住别人的嘴，生气就输了。

**Comment by WenliangCao** [comment]
支持的，这是我用过最稳定的站了呜呜呜，千万要活下去

**Comment by dawancha** [comment]


**Reply by BennyThink** [reply]
这段对话以及下面的对话，犯了以下几种逻辑谬误：
1. 个人攻击
2. 动机谬误
3. 泛化谬误

然后：
1. 你早就同意了6月底的新版 ToS 
2. 此 ToS 已经一个月了
3. 每次我都会发公告，会有等待期，尽管 ToS并没有明确要求这样做
4. 当然是选择和你说再见了！

**Reply by Passerby1011** [reply]
首先我同意一个观点，任何一个站点都是要盈利的要赚钱的。
我认为站点运营思路和方向的转变没有任何问题。
我记得与免费额度有关的 tos 是 6 月才出来的，那 6 月之前的免费额度呢？很多时候用户没有选择的权利，（就和有些公司的条款一样不同意不给用，但这合理吗？）哪些额度是用户一点一点自己积攒出来，不是抢来的，是在你制定的规则下合理获得的，他们也在合理的使用。那些用户实实在在为站点带来了流量和热度，头顶冒火的签到领额度一开始是公益，后期则更像是一种营销手段，但是这个营销你们玩脱了。商业营销，不是你加个条款，就有说啥是啥的

**Reply by Passerby1011** [reply]
不要把什么都归咎于 tos，那玩意有些条款更像是霸王条款，举个不太恰当了栗子，王者荣耀官方说连续登录游玩 100 天说送全皮肤，但是到 99 天说活动取消了，取消就算了，之前送的全部收回，然后指着用户协议说你们同意过的，一切活动的解释权归我所有

**Reply by BennyThink** [reply]
你觉得是营销那就是，你觉得是什么就是什么，就像我上面已经说了的：
> 随便你们怎么说了，说是我拿这个那个帖子做借口也好，说我在怪罪、下台阶、收割韭菜、玩不起等等都好。无所谓，就当我是真的这样想的，就当我觉得猪养肥了要杀了。

我不在乎真的，少了这种类型的用户我会很开心的。

**Reply by Passerby1011** [reply]
有个很有意思的事情，第一我在你的站点付费了虽然不多（使用量也远小于我的付费量，所以本质来说我是个付费用户），第二我来自你说的黑产论坛，第三你上面说所有来自该黑产论坛的用户都将无法访问本站，意思是我的付费额度也要被清除封号吗？乐了矛盾出现了

**Reply by BennyThink** [reply]
根据 referer判断，无法从该论坛跳转

**Comment by dawancha** [comment]


**Reply by zluckyhou** [reply]
活动策略是人家定的，免费用户还要求那么多，你咋不去做慈善呢，是你自己玩不起吧

**Comment by eleanorsubijanodvi62** [comment]
支持，长痛不如短痛，维护付费用户/有价值的用户更能保证长久的运营

**Comment by realnghon** [comment]
这次清空免费用户的免费额度，下次是不是要清付费用户的免费额度了？

要清为什么不全清，说不过去，这里先骗一波付费？

真正付费用户又不会在意你的免费额度，拜拜吧

**Reply by BennyThink** [reply]
尊敬的用户，您使用了怀疑动机和个人攻击的谬误。我理解您的想法，但是我依旧还要声明，付费用户的任何额度都不会动的。是否相信是取决于您，去留也取决去您。

**Reply by sanfy008** [reply]
> 尊敬的用户，您使用了怀疑动机和个人攻击的谬误。我理解您的想法，但是我依旧还要声明，付费用户的任何额度都不会动的。是否相信是取决于您，去留也取决去您。

麻烦可以帮我查看一下我的额度扣除情况吗？

**Reply by BennyThink** [reply]
@sanfy008 好像已经给你解决了，你应该给我发邮件或者私聊了。下次建议你单独发一层，不要在别人的楼层里反馈问题，除非问题很接近。

**Comment by Jacklchen** [comment]
> # TLDR
> 北京时间8月3日零点，将会清空所有**免费用户**的免费额度。所有用户的奖励额度、付费额度、付费用户的任何额度均不受此变动的影响
> 
> # 缘起
> 一切的一切，都源自于这张帖子《不要再被头顶冒火欺骗了，用始皇的 API 服务吧》。
> 
> 当时我们在这件事情上讨论如何处理，最终得出的结论是**转变产品策略**。当时我们进行了一系列不同意见的交换，最终我独裁式的选择缓和的处理方式：逐步处理而不是一刀切的清空免费额度
> 
> # 清零计划
> 昨天进行了一些未活跃用户的清理，具体清理规则可以看这里 #167
> 
> 结果第二天就收到了20多封邮件，还有别人发给我的更让人上火的[某帖子](https://linux.do/t/topic/157312/)😂 （这个帖子被删了，当时大意了忘记存档 不过嘛总有类似[让人上火](https://web.archive.org/web/20240725043249/https://linux.do/t/topic/158667)的东西）
> 
> # 错误
> 在三个星期前做下转型的决定时，我就意识到了，最后无论怎么转型，都是我的错，都会影响站点的口碑和声誉。
> 
> * [用户协议](https://burn.hair/tos)中明确指出，重复注册会被无另行通知的情况下封禁，还是有人说这样做过分
> * 用户协议中说，保留回收公益额度的权利，还有人说恶心人
> * 清零前会写一个帖子，提前几天时间预告，还有人说“说改就改”
> * ……
> 
> 我意识到，即使点了同意，也没有人读用户协议的。
> 
> # 长痛不如短痛
> <img alt="image" width="712" src="https://private-user-images.githubusercontent.com/14024832/351789988-8506bb10-5d23-4248-9848-f64a979d2422.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjE4ODc0NDUsIm5iZiI6MTcyMTg4NzE0NSwicGF0aCI6Ii8xNDAyNDgzMi8zNTE3ODk5ODgtODUwNmJiMTAtNWQyMy00MjQ4LTk4NDgtZjY0YTk3OWQyNDIyLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MjUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzI1VDA1NTkwNVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc1NWJiNmMyZjdiMTk1ZWY2Njc3MGEwZTlmYzUzNzQ2MjQzYjYzYTQ4YWRiZmE2NmE2ZmVhNGE0YTI0NjBkNmImWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.kWSqmqgWQy0corCY7sKIsX-R5t86yCdhrHnrr-NTpYU">
> 这个图是大概是最近几天的消耗，付费用户消耗 259，免费用户消耗662，也就是说我拿了70%的资源用来做可能吃力不讨好的事情。
> 
> 这对于付费用户来说很不公平，而且我也不好保证在这种极度不均衡的情况下，保证付费用户的利益。
> 
> 所以真不如当时听了投资人的意见，长痛不如短痛，一刀切最好。
> 
> > 这个才是重要点的事，三个月都转化不了的用户早点清退掉，给付费用户极致的体验
> > 既然公益影响稳定性，就不搞了，很简单
> 
> 随便你们怎么说了，说是我拿这个那个帖子做借口也好，说我在怪罪、下台阶、收割韭菜、玩不起等等都好。无所谓，就当我是真的这样想的，就当我觉得猪养肥了要杀了。
> 
> # 最终的决定是
> 1. 一周左右的通知期，避免有人说时间太短
> 2. 北京时间8月3日零点（恰巧那个帖子一个月），所有**非付费用户**的免费额度将会被清空
> 3. 付费用户不会受到任何影响，奖励额度也不会有影响
> 4. 所有和这次清零相关的问题，包括邮件、私聊等不会回复
> 5. 签到也将改变为只支持付费用户
> 6. 用户协议推出一月有余，届时所有未同意用户协议的用户都无法使用本服务，只要登录同意即可恢复
> 7. 如果额度分组时有错，请在通知期结束之前告知我查询修改
> 8. 所有来自该黑产论坛的用户都将无法访问本站

玩不起就别玩，菜就多练

**Comment by IceCodeNew** [comment]
做不了太多的事，拍拍 benny
我觉得你做得很对👍

**Comment by yanxici** [comment]
白产哥无敌了

**Comment by pryyxj** [comment]


**Reply by BennyThink** [reply]
你当你是谁呀？😆
<img width="477" alt="image" src="https://github.com/user-attachments/assets/686895ca-5121-4b71-b25d-77975d51b61d">

**Comment by yujingyang0428** [comment]
哎，虽然我很支持，我也每天坚持签到 ，到50多刀了，虽然用的少，但是说清就清了，难受·····

**Comment by PaToSkY** [comment]
为啥充值过几次，也给清了啊😭

**Reply by PaToSkY** [reply]
充值额度用完是不是就不算付费用户了

**Reply by BennyThink** [reply]
对的。为表示你的支持，已经给你恢复一部分了。

**Reply by PaToSkY** [reply]
谢谢

**Comment by magang0425** [comment]
玩不起，哪天付费用户也说清就请了，low

**Comment by arlororo** [comment]
![image](https://github.com/user-attachments/assets/d706cab8-8e42-4fa3-8fa3-ef562f31846b)
我这不算付费用户么，怎么额度都没了。

**Reply by BennyThink** [reply]
因为你已经把这部分额度都用完了，迁移的时候你非付费用户。我给你补了一些查收。

**Reply by arlororo** [reply]
收到了，谢谢

**Comment by guobaozu** [comment]
。。。你这个清理额度的操作，真是让人开眼， 我真是第一次碰到。。。。用的再少，也是别人通过签到得来的。你如果玩不起了，可以发个贴子让大家主动还给你，我相信大家也不缺你那点额度。但你这样随便清理别人名下的额度，这跟耍无赖有何区别？

**Reply by magang0425** [reply]
别bb了，下一步就是封你账号

**Reply by guobaozu** [reply]
抓紧封吧，谢谢。

**Comment by dragonlayout** [comment]
所以最后清理的逻辑并不是之前公告说的那样
> 北京时间8月3日零点，将会清空所有免费用户的免费额度。所有用户的奖励额度、付费额度、付费用户的任何额度均不受此变动的影响

付费用户账户里没有付费额度也会被清除所有免费额度

**Reply by BennyThink** [reply]
也许你理解有误
> 所有持有充值额度的用户均被视为 VIP 用户

用完了就不是了啊

**Comment by dragonlayout** [comment]
[服务条款 （2024年7月1日更新）](https://burn.hair/tos) 
> 3.8 用户等级
所有持有充值额度的用户均被视为 VIP 用户，并且我们可能会根据营销需求去区分不同用户等级，进而提供不同等级的服务承诺。

也没说VIP用户==付费用户吧

**Reply by dragonlayout** [reply]
我知道 TOS 有说
> 公益额度（适用于初次注册、签到、参与活动）：为了防止对免费服务的滥用，本类额度不提供任何稳定性或服务承诺。在资源有限或管理精力不足时，本站保留随时回收公益额度的权利。

可我看到上面有人说自己付费额度被收回了的时候还是震惊了

当然您提供的这个服务确实帮助、方便了很多人，只是这个看到这个确实让人感觉不适

**Reply by BennyThink** [reply]
……

> 可我看到上面有人说自己付费额度被收回了的时候还是震惊了

你再仔细看看，楼上的几个是因为他们把额度用完了，所以当时被算作了普通用户。我全部都给补了一定的额度。邮件里也有很多人反映类似的问题，我都按照每个人的情况补了一些。正常说话的我一个都没漏。

🤷真是让人感到不适。

**Reply by dragonlayout** [reply]
VIP用户==付费用户
那贴下这个解释吧，TOS 里没找到

**Reply by dragonlayout** [reply]
申请退款吧
